contracts/script/test/t1ERC7683PullE2E.s.sol

```diff
@@ -0,0 +1,133 @@
+// SPDX-License-Identifier: MIT
```

Comment:
Please, add this test to https://github.com/t1protocol/t1/pull/90/files#diff-2546446fb782649235f133ea28bca0a6f6df532ae9f7f7123bcd887ccddbef75R81

Reply:
That's "Scripts to test the canonical bridge functionalities". This is an application. Anyway I will find another place for it

Reply:
Feel free to redefine the section.

Reply:
Still missing.

Reply:
fixed